rps.js

```diff
@@ -0,0 +1,81 @@
+
+
+var user= prompt("Do you choose rock, paper or scissors?");
+
+
+
+var computer = Math.floor(Math.random()*3);
+if (computer=== 0) {
+  computerChoice = "rock";
+  }else if(computer === 1) {
+  computer = "paper";
+  }else {
+  computer = "scissors";
+} 
+
+
+
+
+var PlayGame = function(choice1,choice2) {
+
+
+
+if (choice1 === choice2) {
+
+
+
+return " tie!" ;
+
+
+}
+
+
+
+if (choice1 === "rock" && choice2 === "scissors" ) {
+
+
+
+
+return " You Win !!!! "; }
+
+else { return "Computer Wins!! " }
```

Comment:
This will return "Computer Wins!!", before checking out the other conditions, even if the user was winning.
Try it with different choices for the user and see what you get.
How you can make your program check all conditions?